package.json

```diff
@@ -92,23 +92,24 @@
     "!__tests__"
   ],
   "scripts": {
-    "docs": "npm run build && npm run docs:ci",
-    "docs:ci": "npm run docs:extract -- --local && npm run docs:generate",
-    "docs:generate": "api-documenter markdown --input-folder ./tmp --output-folder ./api/docs",
-    "docs:extract": "api-extractor run --verbose",
-    "build": "npm run clean && tsc",
-    "clean": "rm -rf dist",
-    "version:version-file": "genversion --template version-template.ejs --force lib/version.ts",
-    "version": "npm run version:version-file && npm run docs && git add ./lib/version.ts ./api/docs",
-    "test": "npm run test:ts && npm run test:legacy",
-    "test:ts": "jest",
-    "test:legacy": "npm run build -- --declaration false && ./test/scripts/vows.js test/*_test.js",
-    "typecheck": "tsc --noEmit",
-    "cover": "jest --coverage",
-    "lint": "eslint .",
-    "eslint": "eslint .",
-    "prettier": "prettier .",
-    "format": "npm run eslint -- --fix"
+    "build": "npm run _build:clean && npm run _build:compile",
+    "precommit": "npm run build && npm run _api:update && npm run _docs:generate && npm run _format:fix && npm run _lint:fix",
+    "prepack": "npm run build",
+    "test": "npm run build && npm run _test:ts && npm run _test:legacy",
+    "version": "npm run _version:generate && npm run precommit && git add --renormalize .",
+    "_api:check": "api-extractor run --verbose",
+    "_api:update": "api-extractor run --verbose --local",
+    "_build:clean": "rm -rf dist",
+    "_build:compile": "tsc",
+    "_docs:generate": "api-documenter markdown --input-folder ./tmp --output-folder ./api/docs",
+    "_docs:fix": "prettier ./api/docs --write",
```

Comment:
Good question. I found another bug in the `api-documenter` where they aren't respecting the line-endings declared in the configuration. There is already a fix submitted but due to their backlog it's hard to tell when it will be addressed. One suggested workaround is to run the docs through `prettier` after generation since that tool can sort out the wrong line endings in the markdown.